docs/src/main/paradox/routing-dsl/testkit.md

```diff
@@ -215,13 +215,24 @@ The default timeout when testing your routes using the testkit is @scala[1 secon
 In order to extend this default timeout, to say 5 seconds, just add the following implicit in scope:
 
 Scala
-:   @@snip [TestKitFragmentSpec.scala]($test$/scala/docs/http/scaladsl/server/TestKitFragmentSpec.scala) { #timeout-setting }
+:   @@snip [TestKitFragmentSpec.scala](/docs/src/test/scala/docs/http/scaladsl/server/TestKitFragmentSpec.scala) { #timeout-setting }
 
 Java
-:   @@snip [WithTimeoutTest.java]($test$/java/docs/http/javadsl/server/testkit/WithTimeoutTest.java) { #timeout-setting }
+:   @@snip [WithTimeoutTest.java](/docs/src/test/java/docs/http/javadsl/server/testkit/WithTimeoutTest.java) { #timeout-setting }
 
 Remember to configure the timeout using `dilated` if you want to account for slow test systems.
 
+## Testing Actor integration
+
+The @scala[`ScalatestRouteTest`]@java[`JUnitRouteTest`] provides a Classic @apidoc[akka.actor.ActorSystem],
+so if you want to test integration of your route with actors using the new API's you will need to adapt it:
```

Comment:
`the new API's` is a term that will not age well. 😅

I guess you mean typed actor API's?